tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -33,16 +33,16 @@ ${mntNamed}=  /mnt/named
 
 
 *** Keywords ***
-Setup ESX And NFS Suite
+Setup VC And NFS Suite
     [Timeout]    110 minutes
     Run Keyword And Ignore Error  Nimbus Cleanup  ${list}  ${false}
     Log To Console  \nStarting test...
 
-    ${nfs}  ${nfs_ro}  ${esx1}  ${nfs_ip}  ${nfs_ro_ip}  ${esx1_ip}=  Deploy Simple NFS Testbed  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  additional-args=--testbedSpecRubyFile /dbc/w3-dbc302/rashok/vic-nfs.rb --esxBuild ${ESX_VERSION} --esxPxeDir ${ESX_VERSION} --plugin testng
+    ${nfs}  ${nfs_ro}  ${vc}  ${nfs_ip}  ${nfs_ro_ip}  ${vc_ip}=  Deploy Simple VC Cluster With NFS Testbed  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  additional-args=--testbedSpecRubyFile /dbc/sc-dbc1215/rogelios/vic-simple-cluster-with-multiple-nfs.rb --vcvaBuild ${VC_VERSION} --esxBuild ${ESX_VERSION} --esxPxeDir ${ESX_VERSION} --plugin testng
```

Comment:
you should not wrap all of these variables up into additional-args... at most probably just the ruby spec file location var the rest are all standard and should be within the keyword